src/core/crypto.c

```diff
@@ -342,14 +342,14 @@ QuicCryptoReset(
     Crypto->RecoveryEndOffset = 0;
     Crypto->InRecovery = FALSE;
 
-    if (ResetTls) {
```

Comment:
What are the ramifications of not having this?

Reply:
This used to be here because we needed to regenerate the QUIC TP for version negotiation, so that we encoded the received versions (to later authenticate). But since we're not doing that, this is unnecessary. We can just retransmit exiting CRYPTO payload, just like RETRY scenarios.